sources/admin.py

```diff
@@ -16,12 +19,33 @@
 )
 
 
+def get_username(obj):
```

Comment:
How about this for clarity?
```suggestion
def get_user_display_name(obj):
```